sdk/textanalytics/azure-ai-textanalytics/src/test/resources/session-records/TextAnalyticsAsyncClientTest.recognizeLinkedEntitiesForBatchInputShowStatistics[1].json

```diff
@@ -1,26 +1,26 @@
 {
   "networkCallRecords" : [ {
     "Method" : "POST",
-    "Uri" : "https://REDACTED.azure-api.net/text/analytics/v3.1-preview.3//entities/linking?showStats=true&stringIndexType=Utf16CodeUnit",
+    "Uri" : "https://REDACTED.cognitiveservices.azure.com/text/analytics/v3.1-preview.3//entities/linking?showStats=true&stringIndexType=Utf16CodeUnit",
```

Comment:
The earlier endpoint URL was incorrect?

Reply:
Previously, the `url` is not REDACTED, but in the new recording way, the url is REDACTED.

Reply:
https://github.com/Azure/azure-sdk-for-java/pull/18799/files/6ac36b01af96b13eb58cb2361dbe1f3e5e4cf9ce#diff-6a3130cb95edcb0903d9b85ca866245546da97b1e520ae5d747cd00dd425b6baR19